chart/templates/nginx.yaml

```diff
@@ -24,6 +27,12 @@ data:
             server ###ZARF_VAR_BASE_IP###.201:443; 
         }
 
+        {{- if .Values.customGateway | default false}}
+        upstream backend_custom_gateway {
+            server ###ZARF_VAR_BASE_IP###.202:443;
```

Comment:
I think this is a fair assumption, but I believe this assumes that any custom gateway is deployed after admin/tenant (based on the IP allocation). Should be fine, just wanted to note it.